syntax = "proto3";

package products;

service ProductService {
    rpc GetProduct (ProductRequest) returns (ProductResponse);
}

message ProductRequest {
    int32 product_id = 1;
}

message ProductResponse {
    int32 id = 1;
    string name = 2;
    int32 price = 3;
}
